packages/victory-core/src/victory-container/victory-container.js

```diff
@@ -77,40 +69,19 @@ export default class VictoryContainer extends React.Component {
     }
   }
 
-  getChildContext() {
-    return {
-      portalUpdate: this.portalUpdate,
-      portalRegister: this.portalRegister,
-      portalDeregister: this.portalDeregister,
-      getTimer: this.getTimer
-    };
-  }
-
   componentDidMount() {
     if (this.shouldHandleWheel && this.containerRef) {
       this.containerRef.addEventListener("wheel", this.handleWheel);
     }
   }
 
   componentWillUnmount() {
-    if (!this.context.getTimer) {
-      this.getTimer().stop();
-    }
+    this.timer.stop();
```

Comment:
I am not sure whether this is the correct translation here. Should the timer only be stopped by this component if it has created the timer itself?